.github/dependabot.yml

```diff
@@ -10,4 +10,12 @@ updates:
     commit-message:
       prefix: 'fix'
       prefix-development: 'build'
-      include: 'scope'
\ No newline at end of file
+      include: 'scope'
+  - package-ecosystem: "github-actions"
+    directory: "/"
+    schedule:
+      interval: "weekly"
+    commit-message:
+      prefix: 'fix'
```

Comment:
I think both should be `build` for actions - unless it distinguishes / bumps inside action.yml? 🤔 Do you know if it does that?

Reply:
I am not sure it does

Reply:
Mostly i copied the setup from the other setup